test/unit/ldp/operations/HeadOperationHandler.test.ts

```diff
@@ -22,6 +23,6 @@ describe('A HeadOperationHandler', (): void => {
     const result = await handler.handle({ target: { path: 'url' }} as Operation);
     expect(result.identifier.path).toBe('url');
     expect(result.body?.binary).toBe(false);
-    expect(result.body?.data._read(64)).toBeUndefined();
+    await expect(arrayifyStream(result.body?.data ?? streamifyArray([]))).resolves.toEqual([]);
```

Comment:
Why is the `?? streamifyArray([])` part there?

Reply:
Because I cannot pass a possibly undefined readable to arrayifyStream.
And I think having an undefined body.data is fine, because we don't want any data in the HEAD request, so `?? streamifyArray([])` will make the test pass if it should happen.

Reply:
You can use `result.body!.data` to tell TypeScript that you guarantee there is always going to be a body. If you add `?? streamifyArray([])` you test on something the handler is not actually doing (since it will always add an empty body), so I'm ok with this failing if there is no body for now.